### PR TITLE
Update supplier description (1149)

### LINKED ISSUE #1149: Supplier - definition ("awarded or contracted")
In https://github.com/open-contracting/standard/issues/903#issuecomment-728178107, I questioned whether the definition of 'Supplier' ("An entity awarded or contracted to provide goods, works or services.") requires both "awarded" and "contracted".

> As long as the definition includes "awarded", if a buyer awards a contract to supplier A, then supplier B goes to court, then a new award goes to supplier B, you have two suppliers (as long as the original award isn't cancelled, but just doesn't lead to a contract). I'm not sure this is ideal.

> On the other hand, since OCDS only has 'suppliers' in Award, perhaps "awarded" needs to stay as the centrepiece. However, it might be clearer if "contracted" stems out of "awarded", something like "An entity to which the buyer or procuring entity has decided to award a contract".

To clarify, the point of the first paragrah was that if we mention "awarded", then non-intuitive things can sometimes happen. The point of the second paragraph was that maybe having "awarded" is unavoidable, because in OCDS, suppliers are referenced in `Award`, not `Contract`. However, in that case, let's still avoid having two verbs at the same level ("awarded or contracted") and instead have only one verb ("awarded"). Since "awarded" as the only verb might be unclear (e.g. if the definition was "An awarded entity" or "An entity listed in an award"), we should also mention a contract (but not as a verb of its own, but as an object of the main "awarded" verb) and for readability purposes also the subject that awards the contract. Thus the proposed: "An entity to which the buyer or procuring entity has decided to award a contract".

However, thinking about it again and looking at the definition of "Award" in https://github.com/open-contracting/standard/issues/895#issuecomment-720684175 (including https://github.com/open-contracting/standard/issues/895#issuecomment-747835840), I would actually propose to avoid both verbs - "awarding" and "contracting". Instead, let's use a definition reusing the terms from the definition of an award: "An entity with which a buyer or a procuring entity wishes to conclude a contract."

### PR DESCRIPTION
**Pull request by JachymHercher**
closes #1149

### CODE REVIEW

**Comment by JachymHercher** [comment]
Test probaby failed because the codelist does not correctly work with the "deprecated" column. Not sure why though - I thought that was merged into 1.2 through https://github.com/open-contracting/standard/pull/1182? Should be easy to fix when merging.

File: schema/codelists/partyRole.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Code,Title,Description,Deprecated
 buyer,Buyer,"The organization aiming to conclude a contract with a supplier or to use the goods, works or services resulting from the contract.",
 procuringEntity,Procuring entity,"The organization managing the contracting process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",
-supplier,Supplier,"An entity awarded or contracted to provide goods, works or services.",
+supplier,Supplier,"An entity with which a buyer or a procuring entity decided to conclude a contract.",
```
Comment (jpmckinney) [inline schema/codelists/partyRole.csv]: It's a bit silly, but the tests require quotation marks to be omitted unless the cell contains a comma.

```suggestion
supplier,Supplier,An entity with which a buyer or a procuring entity decided to conclude a contract.,
```

**Review by jpmckinney — COMMENTED** [review]
What about the `awards/suppliers` field? Right now it starts with the circular "The suppliers awarded this award".

**Comment by JachymHercher** [comment]
I overlooked it, sorry. The same definition added, just in plural. 

I removed the explanation that was there, as it partially repeats the definition of award ("Depending on the jurisdiction, a single decision may concern a single supplier, item and value; or batches of suppliers and/or items and/or values. [...] As far as possible, the award should be published at the most granular level (i.e. a given supplier will deliver a given item at a given value)" and I think we should rather explain usage in the guidance mentioned in the last paragraph of https://github.com/open-contracting/standard/issues/896#issuecomment-774772868.

File: docs/history/changelog.md [hunk 2]
```diff
@@ -65,6 +66,8 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 * [#1238](https://github.com/open-contracting/standard/pull/1238) Open the `releaseTag` codelist.
 
 ### Schema
+
+* [#1232](https://github.com/open-contracting/standard/pull/1232) Update the description of `award.suppliers`.
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
* [#1232](https://github.com/open-contracting/standard/pull/1232) Update the description of `awards.suppliers`.
```

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
FYI, @yolile This PR essentially overrides #1115. Not sure how important that is.

**Comment by yolile** [comment]
> FYI, @yolile This PR essentially overrides #1115. Not sure how important that is.

The import part of #1115 was to clarify when to use multiple suppliers. We don't have that here in the supplier description anymore, but that guidance will be in the Award definition, as I see in #1175, so  I guess we are good.

**Pull request merged**